tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -18,60 +18,59 @@ import NavItem from '../NavItem/NavItem'
  */
 function HeaderComp() {
   const pathname = useSelector((state) => state.router.location.pathname)
-  const [isMenuVisible, setIsMenuVisible] = useState(false)
   const user = useSelector((state) => state.auth.user)
   const authenticated = useSelector((state) => state.auth.authenticated)
 
   const menuRef = useRef()
 
-  let tabIndex = 0
-  /* istanbul ignore next  */
-  const handleTabKey = (e) => {
-    /* istanbul ignore if */
-    if (menuRef.current.classList.contains('is-visible')) {
-      e.preventDefault()
-      console.log('is visible')
```

Comment:
Is logging now handled elsewhere?

Reply:
Logging to the console like this is generally unnecessary